R/scatter.R

```diff
@@ -670,6 +666,19 @@ volcano_plotter <- function(data, x, p, p_fdr, color,
       scale_x_continuous(limits = x_lim)
   }
 
+  if (!is.null(label)) {
+    if (label %in% colnames(data)) {
+      label_data <- data[data[, p] < label_limit, ]
+      pl <- pl +
+        ggrepel::geom_label_repel(data = label_data,
+                                  mapping = aes(label = label_data[, label]),

```

Comment:
I didn't know this syntax also works! While it does look like handy, I would still change this to the more traditional ggplot style to be consistent with the rest of notame:
```suggestion
                                  mapping = aes_string(label = label),
```